### ISSUE TITLE
Error in rep.int(character, length) : invalid 'times' value

### ISSUE THREAD

**Issue by epruesse** [r-lib/lintr#1953]
```R
> install.packages("lintr")
[...]
> lintr::lint_dir("R")
...................................................
Error in rep.int(character, length) : invalid 'times' value
```

Is there any way to make it tell me at least what file it errored out in? Or which line in the lintr code? Anything? I know this bug report is supremely unhelpful, but there is nothing else I've got.

**Comment by MichaelChirico** [comment]
Sorry about that. Are you using the latest dev version? I thought this error was fixed already.

**Comment by epruesse** [comment]
I was using CRAN. With github main branch version it's telling me this:

```
.Error: Linter 'function_left_parentheses_linter' failed in {some.R}: 
All entries in `ranges` must satisfy 0 <= range[1L] <= range[2L] <= nchar(line) + 1 (28).
```

At least I know which linter broke in which file now. Still wondering which line(s) caused the linter to break though.

**Comment by epruesse** [comment]
Hmm. It seems to be doing that for all my files. Odd.

**Comment by epruesse** [comment]
Ahh! Because of the way emacs indents my R, I'm writing my functions like this:

```R
slack_message <- function
(
     txt,
     ...,
     channel = config::get("slack")$channel,
     username = config::get("slack")$username,
     token = NULL
) {
...
}
```

If I put the `(` on the same line as `function`, the parameters hang so far to the right I'm running out of columns. I suppose I could try and fix the ESS indentation somehow. Still, I suppose `lintr` shouldn't just error out?

**Comment by MichaelChirico** [comment]
thanks! I agree we shouldn't error.

you may be interested in the new double indent style for functions in the tidyverse guide:

https://style.tidyverse.org/functions.html#long-lines-1

**Comment by epruesse** [comment]
That looks almost like what I've been doing. I'm indenting function args same depth as code, giving `) {` it's own line to make separation visually distinctive.

The trick to getting [TheOneTrueEditor](https://xkcd.com/378/) to indent according to double indent style was `(setq-local ess-align-arguments-in-calls '())` (turns off special indenting for arguments in function definitions).

It'll do until we get `black` for R (blackr?)
